sl-namespace-init/templates/eventsource.yaml

```diff
@@ -5,6 +5,18 @@ metadata:
 spec:
   template:
     serviceAccountName: kube-argo
+    {{- with .Values.nodeSelector }}
+    nodeSelector:
+      {{- toYaml . | nindent 6 }}
```

Comment:
you should use indent not nindent ?
    nodeSelector:
      {{- toYaml . | indent 6 }}
or change to 
    nodeSelector: {{- toYaml . | nindent 6 }}

Reply:
Updated